numpy/_core/src/multiarray/convert_datatype.c

```diff
@@ -793,11 +794,11 @@ npy_casting_to_string(NPY_CASTING casting)
 /**
  * Helper function to set a useful error when casting is not possible.
  *
- * @param src_dtype
- * @param dst_dtype
- * @param casting
- * @param scalar Whether this was a "scalar" cast (includes 0-D array with
- *               PyArray_CanCastArrayTo result).
+ * @param src_dtype The source descriptor to cast from
+ * @param dst_dtype The destination type trying to be cast to
+ * @param casting The casting rule that was violated
+ * @param scalar Boolean flag indicating if this was a "scalar" cast (includes 0-D 
+ *               array withPyArray_CanCastArrayTo result).
```

Comment:
```suggestion
 * @param scalar Boolean flag indicating if this was a "scalar" cast.
```
The bracket just got outdated with NumPy 2.